### PR TITLE
Fix tang test (part 2)

### PR DESCRIPTION
**Pull request by jepio**
Fix the tang test by moving the tang listener into the qemu network namespace.

Also an attempt to speed up container build so the second commit will be pushed after the first build completes.

### CODE REVIEW

File: platform/local/cluster.go [hunk 1]
```diff
@@ -93,6 +93,20 @@ func (lc *LocalCluster) GetOmahaHostPort() (string, error) {
 	return net.JoinHostPort(lc.hostIP(), port), nil
 }
 
+func (lc *LocalCluster) NewTangListener() (*net.Listener, error) {
```
Comment (krnowak) [inline platform/local/cluster.go:96]: This does not seem to be specific at all to Tang, so maybe name it `NewListenerInsideClusterNS`?

Comment (jepio) [reply platform/local/cluster.go]: makes sense - pushed the change

**Comment by jepio** [comment]
I ran a test with this PRs container image here: http://jenkins.infra.kinvolk.io:8080/job/container/job/test/20754/console

**Comment by krnowak** [comment]
> I ran a test with this PRs container image here: http://jenkins.infra.kinvolk.io:8080/job/container/job/test/20754/console

Are they still failing?

```
[2024-03-20T10:54:32.376Z] Failed tests:
[2024-03-20T10:54:32.376Z] cl.tang.root
[2024-03-20T10:54:32.376Z] cl.tpm.root
```

**Pull request merged**